src/Form/FormMapper.php

```diff
@@ -137,10 +137,10 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
             }
 
             // NEXT_MAJOR: Remove this block.
-            if (isset($options['help'])) {
+            if (isset($options['help']) && !isset($options['help_html'])) {
                 $containsHtml = $options['help'] !== strip_tags($options['help']);
 
-                if (!isset($options['help_html']) && $containsHtml) {
+                if ($containsHtml) {
```

Comment:
What does this fix?

Reply:
Escape striptag if option label_html is set

Reply:
So is simplest optimization

Reply:
Oh right that makes sense. This should be in a commit saying `Do not strip tags when help_html is set`.

Reply:
Commited